File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        SortedSet<Integer> sortedSet = new TreeSet();
        for (int num : nums) {
            sortedSet.add(num);
        }
        Integer[] sortedNums = sortedSet.toArray(new Integer[0]);
        int longestSequence = 0;
        int currentSequence = 0;
        for (int i = 0; i < sortedNums.length; i++) {
            System.out.println(sortedNums[i]);
            if (i == 0 || sortedNums[i - 1] + 1 == sortedNums[i]) {
                currentSequence++;
            } else {
                currentSequence = 1;
            }
            longestSequence = Math.max(longestSequence, currentSequence);
        }
        return longestSequence;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_count = 0
        i = 0
        while i < len(nums):
            if nums[i] == 1:
                count = 0
                while i < len(nums) and nums[i] == 1:
                    count += 1
                    i += 1
                max_count = max(max_count, count)
            else:
                i += 1
                
        return max_count    